core/schema_utils_test.py

```diff
@@ -484,6 +484,9 @@ def arbitary_method(self, obj: Dict[str, Any]) -> None:
 
         Args:
             obj: dict. Argument which needs to be validated.
+
+        Raises:
+            Exception. If 'Any' type is not the dictionary value type.
```

Comment:
```suggestion
            Exception. If 'Any' type is not the dictionary value type.
```

I don't see any mention of the `Any` type in the code below.

Reply:
I tried correcting the statement. PTAL!!!
Thanks!